### ISSUE TITLE
Symfony profiler - Doctrine panel - "Explain query" broken

### ISSUE THREAD

**Issue by RealJTG** [doctrine/DoctrineBundle#1266]
Explain feature on Doctrine debug panel panel seems to be broken

![Selection_070](https://user-images.githubusercontent.com/930399/100246376-b9b5a100-2f41-11eb-8c1c-2be3aeaa6c75.png)

GET /_profiler/be783a?panel=db&page=explain&connection=test&query=3
Trace:
```
TypeError:
Argument 2 passed to Doctrine\Bundle\DoctrineBundle\Controller\ProfilerController::explainSQLServerPlatform() must be of the type string, array given, called in /www/test/vendor/doctrine/doctrine-bundle/Controller/ProfilerController.php on line 66

  at /www/test/vendor/doctrine/doctrine-bundle/Controller/ProfilerController.php:102
  at Doctrine\Bundle\DoctrineBundle\Controller\ProfilerController->explainSQLServerPlatform(object(Connection), array('sql' => 'SELECT COUNT(*) FROM *** WHERE StateID = :dcValue1', 'params' => object(Data), 'types' => array('dcValue1' => 1073741824), 'executionMS' => 0.05746006965637207, 'explainable' => true, 'runnable' => true))
     (/www/test/vendor/doctrine/doctrine-bundle/Controller/ProfilerController.php:66)
  at Doctrine\Bundle\DoctrineBundle\Controller\ProfilerController->explainAction('be783a', 'tools', array('sql' => 'SELECT COUNT(*) FROM *** WHERE StateID = :dcValue1', 'params' => object(Data), 'types' => array('dcValue1' => 1073741824), 'executionMS' => 0.05746006965637207, 'explainable' => true, 'runnable' => true))
...
```

Version:
```
"name": "doctrine/doctrine-bundle",
"version": "2.2.1",
"source": {
    "type": "git",
    "url": "https://github.com/doctrine/DoctrineBundle.git",
    "reference": "9e07bb1ff35d35d9ec4597b79e5d05502d7d4d43"
},
```

**Comment by stof** [comment]
Looks like the typehint added on `explainSQLServerPlatform` for `$query` is wrong. It has been typehinted as `string` in DoctrineBundle 2, while it is an `array` (like for the explain methods of other platforms)

**Comment by RealJTG** [comment]
Yep, broken here https://github.com/doctrine/DoctrineBundle/commit/50c53bef10617dc3c3e8e279f3be6b4e38abc8f6
I'll send PR.

**Comment by ostrolucky** [comment]
Will be fixed in 2.2.2
